spring-cloud-stream-test-support/src/test/java/org/springframework/cloud/stream/test/example/ExampleTest.java

```diff
@@ -39,8 +39,10 @@
  * correctly.
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = ExampleTest.MyProcessor.class, properties = { "server.port=-1", "--spring.cloud.stream.bindings.input.contentType=text/plain", "--spring.cloud.stream.bindings.output.contentType=text/plain",
-		"--spring.main.allow-bean-definition-overriding=true"})
+@SpringBootTest(classes = ExampleTest.MyProcessor.class, webEnvironment = SpringBootTest.WebEnvironment.NONE,
+		properties = {
+		"--spring.cloud.stream.bindings.input.contentType=text/plain",
+		"--spring.cloud.stream.bindings.output.contentType=text/plain"})
```

Comment:
I think we need to remove these `--`. That's not command line arg, but key/value props.
Otherwise let's just remove them as a noise since it works even with them, wrongly provided.

Or am I missing anything and it really works like a charm even in this installment?

Thanks

Reply:
ya, tests passed but will look.